erpnext/projects/doctype/activity_cost/activity_cost.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe import _
+from frappe.model.document import Document
+
+class ActivityCost(Document):
+	def validate(self):
+		self.set_title()
+		self.check_unique()
+		
+	def set_title(self):
+		self.title = _("{0} for {1}").format(self.employee_name, self.activity_type)
+		
+	def check_unique(self):
```

Comment:
This will crash when you try this the second time. You need to exclude current name in filters.